libs/partners/ibm/pyproject.toml

```diff
@@ -0,0 +1,96 @@
+[tool.poetry]
+name = "langchain-ibm"
+version = "0.0.1"
+description = "An integration package connecting IBM watsonx.ai and LangChain"
+authors = ["IBM"]
+readme = "README.md"
+repository = "https://github.com/langchain-ai/langchain"
+license = "MIT"
+
+[tool.poetry.urls]
+"Source Code" = "https://github.com/langchain-ai/langchain/tree/master/libs/partners/ibm"
+
+[tool.poetry.dependencies]
+python = ">=3.10,<4.0"
+langchain-core = ">=0.1.13,<0.2"
+ibm-watsonx-ai = ">=0.1.4"
```

Comment:
Does this package follow semver? Should we make it `^0.1.4` instead?

Reply:
I agree, Changed